docs/cosmology/index.rst

```diff
@@ -108,8 +108,7 @@ Using `astropy.cosmology`
 More detailed information on using the package is provided on separate pages,
 listed below.
 
-.. toctree::
-   :maxdepth: 1
+.. contents::
```

Comment:
Gives error:

```
docs/cosmology/index.rst:111: ERROR: Error in "contents" directive:
no content permitted.

.. contents::

   io/index
   Units and Equivalencies <units>
   For Developers <dev>
```

Reply:
`docs/visualization` also has a sub-directory with its own index. Maybe you can compare between this and that one and see...

https://docs.astropy.org/en/latest/visualization/index.html